src/components/pages/About.js

```diff
@@ -41,7 +41,7 @@ export default class About extends Component {
                     </div>
 
                     <div className="rpgui-center">
-                        <a href="https://github.com/h34d"
+                        <a href="https://github.com/craeftgame/craeft"
```

Comment:
sorry but the code is already in line 63: https://github.com/craeftgame/craeft/pull/3/files#diff-13c98b5e12af3a490a7c7d1a422d49daR63

this is the link to my GitHub profile